include/SH3/graphics/texture.hpp

```diff
@@ -1,56 +1,89 @@
-/*++
+/** @file
+ *
+ *  Functions and structures to load a texture from an arc section
+ *
+ *  \copyright 2016  Palm Studios
+ *
+ *  \note   I'd like to thank Mike for all this, he put a lot of time into reverse engineering
+            all of the file types, including all of the textures, meshes and even the motion capture
+            skeletal animation that Konami captured. Thanks mate!
+ *
+ *  \date 2/1/2017
+ *
+ *  \author Jesse Buhagiar
+ */
+#ifndef SH3_TEXTURE_H_INCLUDED
+#define SH3_TEXTURE_H_INCLUDED
 
-Copyright (c) 2016  Palm Studios and Mike M (@perdedork)
+#include <SH3/arc/types.hpp>
 
-Module Name:
-        sh3_texture.hpp
+#include <cstdint>
+#include <vector>
+#include <GL/gl.h>
 
-Abstract:
-        Defines data structures to load an manipulate a texture
+namespace sh3_graphics
+{
+    /**
+     * Full texture header (both batch and individual texture).
```

Comment:
Add an empty comment line between `Full texture header` and the rest, because the first line is treated as a brief description by Doxygen.